driver/bpf/fillers.h

```diff
@@ -1941,6 +1941,14 @@ static __always_inline int bpf_accumulate_argv_or_env(struct filler_data *data,
 		}
 
 		len = bpf_probe_read_user_str(&data->buf[off & SCRATCH_SIZE_HALF], SCRATCH_SIZE_HALF, arg);
```

Comment:
This is related to this https://github.com/falcosecurity/libs/issues/866, I would expect `bpf_probe_read_user_str` to return `1` when the string is empty but indeed it returns `0` not considering the NULL terminator... BTW your workaround makes sense